### ISSUE TITLE
task: create Doc site for PoC

### ISSUE THREAD

**Issue by slawr** [COVESA/cdsp#9]
Documentation is an important feature of the CDSP. Create a Documentation website for the initial CDSP PoC.

Rough task breakdown:

- [x] decide what doc framework to use
- [x] create doc structure in the sources
- [x] automate website generation if possible (possibly postpone if a lot of work until the framework is confirmed as the one going forward)

**Comment by slawr** [comment]
Decision was taken to use the Hugo static website generation tooling used in other COVESA projects as a starting point.
@UlfBj volunteered to create the initial Hugo implementation. Adding him as an assignee to this issue for the related tasks.

**Comment by slawr** [comment]
@UlfBj has created a PR #17 to add the Hugo infrastructure.
He's waiting on me to work with him to integrate automation of the document generation into the github workflow as I have the access rights.

**Comment by slawr** [comment]
Updated Hugo commits were merged.
Github workflow has been implemented for CI to build the online documentation and host it in github pages.

**Comment by slawr** [comment]
Moved the two sub-tasks for creation of content into new issue #24. This ticket was related to creation of the Doc site which is now live. As it has been open for some time it makes sense to close this and deal with content creation as a follow on task
